src/urllib3/fields.py

```diff
@@ -68,60 +81,89 @@ def format_header_param_rfc2231(name: str, value: Union[str, bytes]) -> str:
     return value
 
 
-_HTML5_REPLACEMENTS = {
-    "\u0022": "%22",
-    # Replace "\" with "\\".
-    "\u005C": "\u005C\u005C",
+_header_value_quote_map = {
```

Comment:
In this case it's probably more performant since this `re.sub` is calling a function for each match. I'll leave it to someone else to figure out how to figure out if `str.replace` is even faster given the typical values being processed.